android/app/src/test/java/org/smartregister/fhircore/StructureMapPlaygroundUtils.kt

```diff
@@ -29,8 +29,14 @@ import org.junit.Test
 import org.robolectric.annotation.Config
 import org.smartregister.fhircore.shadow.FhirApplicationShadow
 
+/**
+ * Provides a playground for quickly testing and authoring questionnaire and the respective
+ * StructureMap
+ *
+ * This should be removed at a later point once we have a more clear way of doing this
+ */
 @Config(shadows = [FhirApplicationShadow::class])
-class StructureMapUtils : RobolectricTest() {
+class StructureMapPlaygroundUtils : RobolectricTest() {
```

Comment:
what about `StructureMapUtilitiesTest`?

Reply:
It's not a test per se, so I thought the naming and class documentation should help make that clear and prevent any confusion

Reply:
oh I see, these are utilities and fixtures? Then maybe `StructureMapTestUtilities`?

Reply:
Yea, this change has been made